freedomofpress/securedrop
# Update developer docs for 0.4 (Part 1)

## Status

Ready for review

## Description of Changes

Partial satisfaction of dev docs updates described in #1208. Submitting these changes to keep the diffs light and aid in review.

For detailed descriptions of the changes proposed in this pull request, please see the commit messages. I made sure to include both explanations of the change and the motivation to implement it. (If you disagree, holler.)

## Testing

Review the diff to ascertain which documents have changed, then build the docs locally and follow the updated instructions.

* Flip between this feature branch and `develop` with livereload active, and see which version of the code blocks you like more.
* Create a new virtualenv and use the `securedrop/requirements/develop-requirements.txt` file to set it up. Confirm you can:
  * build the docs
  * provision the develop VM
  * run the config tests
  * run the app tests (from inside the development VM)

If so, then we're set.

## Deployment

No special considerations for deployment; these changes are strictly to docs and to a lesser extent the dev environment (via the new requirements file).

## Checklist
N/A on testing.
